### PR TITLE
Step-10A: Style Task Card

### PR DESCRIPTION
**Pull request by sta-wtag**
### **[Step-10A](https://trello.com/c/1tRkXyUL/8-step-10a-style-task-card)**

Add styling to Task Card.

### Acceptance Criteria

- [x] Add styling to Task cards according to [design](https://www.figma.com/file/9Rwb2k9Dfbh8Ycb6kbJfzZ/Todo-App-UI-for-New-Joiner?node-id=0%3A1&t=fqQdwByuj6siGx5y-0).

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -150,3 +158,41 @@ textarea {
...
+  border-radius: 5px;
+}
+.swal-text {
+  font-family: 'Roboto';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 21px;
+  margin:0px;
+  color:White
```
Comment (hhu-wtag) [inline assets/css/main.css:201]: Typo? 
```suggestion
  color: white
```

File: components/TaskCard.vue [hunk 2]
```diff
@@ -159,6 +171,14 @@ export default {
 
       this.showEditIcon = true;
       this.loading = false;
+      this.alert();
+    },
+    alert() {
+      swal('Changes are saved successfully', {
```
Comment (hhu-wtag) [inline components/TaskCard.vue]: Please use i18n here

File: components/TaskContainer.vue [hunk 3]
```diff
@@ -114,6 +113,12 @@ export default {
       if (this.taskDescription.length <= 0) {
         this.titleInputError = true;
         this.titleErrorMsg = 'Field is empty';
+        swal('Field is empty', {
```
Comment (hhu-wtag) [inline components/TaskContainer.vue]: Ditto

File: components/TaskContainer.vue [hunk 4]
```diff
@@ -123,6 +128,12 @@ export default {
     addTask() {
       this.$store.dispatch('todos/addTask', this.taskDescription);
       this.$store.dispatch('todos/setTotalPage');
+      swal('Changes are saved successfully', {
```
Comment (hhu-wtag) [inline components/TaskContainer.vue]: Ditto

File: helpers/helper.js [hunk 5]
```diff
@@ -8,7 +8,9 @@ const helper = {
     return formatDistance(date2, date1, { addSuffix: true });
   },
   formatDate(val) {
-    return format(new Date(val), 'dd.MM.yyyy');
+    if (val) {
+      return format(new Date(val), 'dd.MM.yyyy');
+    }
```
Comment (hhu-wtag) [inline helpers/helper.js:22]:
```suggestion
    return val ? format(new Date(val), 'dd.MM.yyyy') : undefined
```

**Review by hhu-wtag — APPROVED** [review]

**Review by sai-wtag — APPROVED** [review]
Locally tested. LGTM. Nice Work. Thanks

File: assets/css/main.css (hunk 1 above)
Comment (sta-wtag) [reply assets/css/main.css:201]: Resolved

File: components/TaskCard.vue (hunk 2 above)
Comment (sta-wtag) [reply components/TaskCard.vue]: resolved

**Review by bia-wtag — CHANGES_REQUESTED** [review]
Tested locally. Could you please have a look at the following issues?

- Task card style is broken for long titles. Please use a max-width along with consideration for a long title.
Screenshot 2023-02-26 at 10 44 26 AM

- Layout and card style broken for mobile, that is, it's scrollable vertically.
Screenshot 2023-02-26 at 10 47 31 AM

**Review by bia-wtag — APPROVED** [review]

**Pull request merged**